schemas/isd.json

```diff
@@ -0,0 +1,242 @@
+{
+   "$schema": "http://json-schema.org/draft-07/schema#",
+   "title": "Instrument Specification Definition",
+   "description": "The Instrument Specification Definition for Astro CSM",
+   "type": "object",
+
+   "properties": {
+       "BORESIGHT": {
+	   "description": "Boresight (x,y,z) vector",
+           "type": "array",
+	    "items": {
+                 "type": "number",
+		 "description": "Component of the boresight vector",
+		 "minItems": 3,
+                 "maxItems": 3
+            }
+       },
+       "DETECTOR_CENTER": {
+           "description": "Center of the detector in pixel coordinates",
```

Comment:
What is the order of the pixel coordinates? Line, Sample?

Reply:
If this is undecided, we should go with line, sample because that's the CSM standard.

Reply:
:+1: Let's do that, then.